pom.xml

```diff
@@ -29,7 +29,7 @@
     </parent>
 
     <artifactId>chronicle-bytes</artifactId>
-    <version>2.24ea17-SNAPSHOT</version>
+    <version>2.24ea18-SNAPSHOT</version>
```

Comment:
2.24ea17 seems to have gone missing

Reply:
I think we made a release yesterday so maybe need to rebase